crates/ntx-builder/src/transaction.rs

```diff
@@ -123,7 +123,8 @@ impl NtxContext {
                 let notes = notes.into_iter().map(Note::from).collect::<Vec<_>>();
                 let (successful, failed) = self.filter_notes(&data_store, notes).await?;
                 let executed = Box::pin(self.execute(&data_store, successful)).await?;
-                let proven = Box::pin(self.prove(executed.into())).await?;
+                let tx_inputs = executed.into_parts().0;
```

Comment:
Might be worth making `into_tx_inputs()` given we do this 3+ times now.

Reply:
That's a change that'd have to be done in `miden-base` iiuc